content/en/docs/collector/trace-receiver.md

```diff
@@ -275,15 +274,14 @@ func (cfg *Config) Validate() error {
 }
 ```
 
-> #### Check your work
->
-> - I imported the `fmt` package, so I can properly format print my error
->   messages.
-> - I added the `Validate` method to my Config struct where I am checking if the
->   `interval` setting value is at least 1 minute (1m) and if the
->   `number_of_traces` setting value is greater or equal to 1. If that is not
->   true the Collector will generate an error during its startup process and
->   display the message accordingly.
+{{% alert title="Check your work" color="info" %}}
+- I imported the `fmt` package, so I can properly format print my error messages.
+- I added the `Validate` method to my Config struct where I am checking if the
+ `interval` setting value is at least 1 minute (1m) and if the
+ `number_of_traces` setting value is greater or equal to 1. If that is not
+ true the Collector will generate an error during its startup process and
+ display the message accordingly. {{% /alert %}}
+
```

Comment:
Overall, this is looking better. Make sure that each `{{% %}}` tag is on its own line. Here is an example, change all other instances too:

```suggestion
{{% alert title="Check your work" color="info" %}}

- I imported the `fmt` package, so I can properly format print my error messages.
- I added the `Validate` method to my Config struct where I am checking if the
 `interval` setting value is at least 1 minute (1m) and if the
 `number_of_traces` setting value is greater or equal to 1. If that is not
 true the Collector will generate an error during its startup process and
 display the message accordingly.
 
 {{% /alert %}}

```

Reply:
`color="primary"` is a valid color code, check docsy - https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers

Also, I have formatted the alerts sections as per your suggestions.